sample-code/spring-app/pom.xml

```diff
@@ -189,6 +188,30 @@
           <skipTests>${skipTests}</skipTests>
         </configuration>
       </plugin>
+      <!-- Skip release -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>deploy-file</goal>
```

Comment:
(Minor/Question)

If I understood correctly you are skipping both for `<goals><goal>deploy-file` and `<goals/>`. Isn't the former redundant? I'm not sure when it comes to Maven plugin configurations.

Reply:
During my testing I found that both needed to be present to correctly skip.